testing/chain_test.go

```diff
@@ -23,11 +23,11 @@ func TestChangeValSet(t *testing.T) {
 	amount2, ok := sdk.NewIntFromString("30000000000000000000")
 	require.True(t, ok)
 
-	val := chainA.App.GetStakingKeeper().GetValidators(chainA.GetContext(), 4)
+	val := chainA.GetSimApp().StakingKeeper.GetValidators(chainA.GetContext(), 4)
```

Comment:
Is it not possible to keep using `App.GetIBCTestStakingKeeper()` and add `GetValidators` and `Delegate` to the `StakingKeeper` interface?

Reply:
i think the idea is that in our simApp, we would continue using the staking module's staking keeper so we don't actually need to add these two methods as they are already there. 

it's just that in the testing chain and testing app setup, we would like to enable osmosis to swap out their customised interfluid staking keeper so they can use our testing setup. this is why we only need that one method which is used in the testing/chain.go file.

Reply:
I actually requested those functions to not be added. I think we should keep our interfaces as small as possible (ideally we wouldn't even need the staking keeper interface)